docs/ehrql-tutorial.md

```diff
@@ -2,7 +2,50 @@
 
 ---8<-- 'includes/data-builder-danger-header.md'
 
-This page will contain an [ehrQL](ehrql-intro.md) tutorial. It will
-initially describe how a relatively minimal ehrQL example works.
+In this tutorial, we will walk through a minimal ehrQL example.
+The example will create a dataset of all patients who were born in or before 2000.
+We will walk through the example step-by-step;
+if you want to see the example in full, then you can [skip to the end](#wrapping-up).
+
+First, we import the patients table and create a variable that references each patient's year of birth:
```

Comment:
I've not used variable, method, function etc. names in the body, as they are more brittle than the associated concepts. That is, `patients` is more brittle than "the patients table".